gitlfs-common/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 description = "Java Git-LFS common structures"
 
 dependencies {
-    compile("com.fasterxml.jackson.core:jackson-databind:2.9.9.3")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.9.9.3")
```

Comment:
Using `api("com.fasterxml.jackson.core:jackson-databind:2.9.9.3")` here and adding `apply(plugin = "java-library")` to the root project's build file resolves the dependency issue.
`